### PR TITLE
Provide devfile for ready to edit workspace for che-docs

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Add a devfile.yaml for editing and implementing the documentation of eclipse che

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14196

### CODE REVIEW

**Comment by olexii4** [comment]
depends on  https://github.com/eclipse/che-plugin-registry/pull/218

**Comment by themr0c** [comment]
For `livereload` you probably want to additionnally open port: 35729.

**Review by mmorhun — APPROVED** [review]

**Comment by olexii4** [comment]
@themr0c  I have fixed. Thanks

File: devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        command: >-
+          cd /projects/che-docs/src/main && mkdir -p _site && jekyll build
+          --config _config.yml,_config-war.yml
+        component: che-docs-dev
+  - name: serve che-docs
+    actions:
+      - workdir: /projects/che-docs/src/main
+        type: exec
+        command: >-
+          cd /projects/che-docs/src/main && jekyll serve --incremental
```
Comment (AndrienkoAleksandr) [inline devfile.yaml]: Sorry, you already set up working dir, I think you don't need extra command: `cd /projects/che-docs/src/main &&`: `cd /projects/che-docs/src/main &&`

File: devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - id: joaompinto/vscode-asciidoctor/latest
+    type: chePlugin
+apiVersion: 1.0.0
+commands:
+  - name: build che-docs
+    actions:
+      - workdir: /projects/che-docs/src/main
+        type: exec
+        command: >-
+          cd /projects/che-docs/src/main && mkdir -p _site && jekyll build
```
Comment (AndrienkoAleksandr) [inline devfile.yaml]: The same, extra command: `cd /projects/che-docs/src/main &&`

**Comment by olexii4** [comment]
@rkratky @slemeur  WDYT? Can I merge this PR?

**Review by AndrienkoAleksandr — APPROVED** [review]

**Comment by mmorhun** [comment]
@olexii4 please add into the PR description which devfile you want to add

**Review by slemeur — APPROVED** [review]

**Pull request merged**